integrations/tensorflow/bindings/python/pyiree/tf/support/module_utils.py

```diff
@@ -197,24 +150,21 @@ def _incrementally_compile_tf_signature_def_saved_model(
     A compiled IREE module blob and the path to the compiled VM FlatBuffer if
     artifacts_dir is provided.
   """
-
-  def _compile_module(saved_model_dir, saved_model_tags, backend_info,
-                      exported_name, artifacts_dir):
-    # Convert the tf_module into raw TF input MLIR.
-    compiler_module = compiler.tf_signature_def_saved_model_to_compiler_module(
-        saved_model_dir, saved_model_tags, [exported_name], pass_pipeline=())
-    return _incrementally_lower_compiler_module(compiler_module, backend_info,
-                                                artifacts_dir)
-
-  # Avoid the crash reproducer under tests or if the flag is false.
-  # Developers can run tests outside of the test runner (e.g. `bazel run`) to
-  # use the crash reproducer.
-  if (FLAGS.capture_crash_reproducer and not _running_bazel_test()):
-    _compile_module = _setup_mlir_crash_reproducer(_compile_module,
-                                                   artifacts_dir,
-                                                   backend_info.backend_id)
-  return _compile_module(saved_model_dir, saved_model_tags, backend_info,
-                         exported_name, artifacts_dir)
+  output_kwargs = (_get_tf_import_output_kwargs(
+      artifacts_dir, backend_info.backend_id) if artifacts_dir else {})
```

Comment:
Looks like this one is short enough that it could follow  https://google.github.io/styleguide/pyguide.html#2114-decision, but `yapf` disagrees with their formatting.